terraform/deploy-static-to-s3.sh

```diff
@@ -0,0 +1,48 @@
+#!/bin/sh
+# Usage deploy-static-to-s3.sh <name of s3 bucket> <build number>
+# eg deploy-static-to-s3.sh nice-cks-local-s3-web-hosting 123
+echo "Deploying files to S3 bucket: $1 into folder $2"
+
+# cache-control: public, max-age=0, must-revalidate
+aws s3 cp ../gatsby/public/ s3://$1/$2 \
```

Comment:
Can use a parameter for the gatsby path? e.g.
```suggestion
$gatsbyStaticSitePath=$1
aws s3 cp $gatsbyStaticSitePath s3://$1/$2 \
```